sktime/utils/plotting.py

```diff
@@ -2,17 +2,42 @@
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 
-__all__ = ["plot_series"]
-__author__ = ["Markus Löning"]
+__all__ = ["plot_series", "check_pred_int"]
+__author__ = ["Markus Löning", "Ifeanyi Eze"]
 
 import numpy as np
 
 from sktime.utils.validation._dependencies import _check_soft_dependencies
 from sktime.utils.validation.forecasting import check_y
+import pandas as pd
 
 
-def plot_series(*series, labels=None, markers=None):
-    """Plot one or more time series
+def check_pred_int(pred_int):
```

Comment:
We should move this file to `sktime.utils.validation.forecasting`

Reply:
Do you mean the validation folder? Because I don't think it's possible to move the plotting.py file to forecasting.py

Reply:
I mean we should move the function `check_pred_int` into `sktime.utils.validation.forecasting.py` and then import here